setcap.sh

```diff
@@ -11,14 +11,17 @@
 if [ -x "$AVB_SETCAP_TOOL" ]; then
 	echo "Using setcap_tool defined on AVB_SETCAP_TOOL env var"
 	$AVB_SETCAP_TOOL $1 && exit
-	echo "Failed"
+	echo "Failed to set cap"
 fi
-
-which setcap_tool &> /dev/null
-if [ $? -eq 0 ]; then
+if [ `which setcap_tool &> /dev/null ` ]; then
 	echo "Trying to use setcap_tool on PATH env var"
 	setcap_tool $1 && exit
-	echo "Failed"
+	echo "Failed to set cap"
+fi
+if [ `which setcap` ]; then
```

Comment:
Oh, before you merge, could you add `&> /dev/null` here, so that it doesn't pollute output?